environs/bootstrap/bootstrap.go

```diff
@@ -751,11 +751,13 @@ func finalizeInstanceBootstrapConfig(
 	}
 
 	icfg.Bootstrap.StateServingInfo = controller.StateServingInfo{
-		StatePort:    controllerCfg.StatePort(),
-		APIPort:      controllerCfg.APIPort(),
-		Cert:         string(cert),
-		PrivateKey:   string(key),
-		CAPrivateKey: args.CAPrivateKey,
+		StatePort:          controllerCfg.StatePort(),
+		APIPort:            controllerCfg.APIPort(),
+		GrpcAPIPort:        controllerCfg.GrpcAPIPort(),
```

Comment:
I don't think this is required in `StateServingInfo`. It is only used to write agent config informing communication with the API server. At this stage agents won't use gRPC.

Reply:
OK